### PR TITLE
Added Zenodo citation

### PR DESCRIPTION
**Pull request by ml-evs**
Another PR, following https://github.com/Materials-Consortia/OPTIMADE/pull/332

I moved a sentence around on the homepage too.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -24,6 +27,6 @@ existing and hypothetical materials, many of which have appeared online:
 - [the Joint Automated Repository for Various Integrated Simulations](https://jarvis.nist.gov)
 - ...
 
-The **Open Databases Integration for Materials Design** (OPTIMADE) consortium
-aims to make materials databases interoperational by developing a common REST API.
+Should you wish to cite the OPTIMADE specification, please use
 
+> Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195051](http://doi.org/10.5281/zenodo.4195051)
```
Comment (ml-evs) [inline index.md:32]: Pending discussion in the spec PR, we will decide whether to link to the unversioned DOI (the suggestion below) or keep it at v1.

```suggestion
> Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](http://doi.org/10.5281/zenodo.4195050)
```

Comment (merkys) [reply index.md:32]: Maybe HTTPS instead of HTTP here too?

**Review by rartino — APPROVED** [review]
Looks good to me.

**Comment by rartino** [comment]
Since we only require a single approval for the website changes, I went ahead and merged this. But now that I see it in place on the website, I wonder if the citation shouldn't have been: 
> Andersen *et al*, The OPTIMADE Specification, [DOI:10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050)
Note the prefix `DOI:`, or alternatively: `https://doi.org/`

**Comment by ml-evs** [comment]
> Since we only require a single approval for the website changes, I went ahead and merged this. But now that I see it in place on the website, I wonder if the citation shouldn't have been:
> 
> > Andersen _et al_, The OPTIMADE Specification, [DOI:10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050)
> 
> Note the prefix `DOI:`, or alternatively: `https://doi.org/`

The link itself goes to https://doi.org/<DOI>, feel free to change the visual display to that too

**Pull request merged**